tests/defaults.rs

```diff
@@ -0,0 +1,430 @@
+use httpmock::Method::GET;
+use httpmock::{Mock, MockRef, MockServer};
+use std::io::{self, BufRead};
+use std::thread;
+
+mod common;
+
+use goose::prelude::*;
+
+const INDEX_PATH: &str = "/";
+const ABOUT_PATH: &str = "/about.html";
+
+const USERS: usize = 3;
+const RUN_TIME: usize = 2;
+const HATCH_RATE: usize = 10;
+const LOG_LEVEL: usize = 0;
+const METRICS_FILE: &str = "metrics-test.log";
+const DEBUG_FILE: &str = "debug-test.log";
+const LOG_FORMAT: &str = "raw";
+const THROTTLE_REQUESTS: usize = 10;
+
+// Has tests:
+// - GooseDefault::Host
+// - GooseDefault::MetricsFile
+// - GooseDefault::MetricsFormat
+// - GooseDefault::DebugFile
+// - GooseDefault::DebugFormat
+// - GooseDefault::HatchRate
+// - GooseDefault::RunTime
+// - GooseDefault::ThrottleRequests
+// - GooseDefault::Users
+// - GooseDefault::NoResetMetrics
+// - GooseDefault::StatusCodes
+// - GooseDefault::OnlySummary
+// - GooseDefault::NoTaskMetrics
+// - GooseDefault::Manager
+// - GooseDefault::ExpectWorkers
+// - GooseDefault::NoHashCheck
+// - GooseDefault::ManagerBindHost
+// - GooseDefault::ManagerBindPort
+// - GooseDefault::Worker
+// - GooseDefault::ManagerHost
+// - GooseDefault::ManagerPort
+
+// Can't be tested:
```

Comment:
@LionsAd Note that we can't test LogFile, Verbose, or LogLevel in a meaningful way because of a limitation of the logging library we're using -- I was going to explore if there are other/better libraries out there that remove this limitation.